### PR TITLE
net-misc/oidentd: add EAPI6 ebuild with more masquerading patches and selinux support

### PR DESCRIPTION
**Pull request by Dessa**

### CODE REVIEW

**Comment by Dessa** [comment]
one thing to note is that after testing i realized that CONFIG_INET_TCP_DIAG is required in the kernel nowadays to work (at all; at least in my environment), i didn't add a check for that since its normally default, but maybe that is wrong?

**Comment by mgorny** [comment]
> one thing to note is that after testing i realized that CONFIG_INET_TCP_DIAG is required in the kernel nowadays to work (at all; at least in my environment), i didn't add a check for that since its normally default, but maybe that is wrong?

Well, if you can normally disable it, then you should probably add a check for it. Make it non-fatal (I.e. `~foo`) since it's needed only at runtime.

Also, would you be interested in taking over maintenance of that package?

File: net-misc/oidentd/files/oidentd-2.0.8-masquerading.patch [hunk 1]
```diff
@@ -1,4 +1,4 @@
---- oidentd.orig/src/kernel/linux.c	2006-05-22 06:58:53.000000000 +0300
+--- oidentd-2.0.8/src/kernel/linux.c	2006-05-22 06:58:53.000000000 +0300
```
Comment (mgorny) [inline net-misc/oidentd/files/oidentd-2.0.8-masquerading.patch]: Why are you changing this? The old form should be perfectly fine for -p1.

File: net-misc/oidentd/files/oidentd-2.0.8-bind-to-ipv6-too.patch [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Patch to bind to ipv6 socket as well
 Patch supplied by Fabian Knittel <fabian.knittel@avona.com>
---- oidentd-2.0.8//src/oidentd_inet_util.c	2006-05-22 02:31:19.000000000 +0200
```
Comment (mgorny) [inline net-misc/oidentd/files/oidentd-2.0.8-bind-to-ipv6-too.patch]: I suppose the only problem here would be the `//`.

**Comment by Dessa** [comment]
the reason i did the prep commit for EAPI 6 is touching stable. i initially thought that'd be a good idea but i can see now that it is not really helpful.

File: net-misc/oidentd/files/oidentd-2.0.8-bind-to-ipv6-too.patch (hunk 2 above)
Comment (Dessa) [inline net-misc/oidentd/files/oidentd-2.0.8-bind-to-ipv6-too.patch]: github for some reason doesn't see the change, but it is done

Comment (mgorny) [inline net-misc/oidentd/files/oidentd-2.0.8-bind-to-ipv6-too.patch]: It sees it, don't worry.

**Pull request closed without merging**